autogluon/utils/tabular/features/generators/abstract.py

```diff
@@ -571,6 +669,7 @@ def print_generator_info(self, log_level: int = 20):
         """
         if self.fit_time:
             self._log(log_level, f'\t{round(self.fit_time, 3)}s\t= Fit runtime')
+            self._log(log_level, f'\t({len(self.features_in)}, {len(self.features_out)}) = (input feature count, output feature count)')
```

Comment:
I worry some novices may confuse "fit" with model-fitting and would hope the output remains clear to somebody who has no idea what a "FeatureGenerator" is.  I think this would be clearer if printed as for example:
```
Fitting AutoMLPipelineFeatureGenerator to preprocess the data...
```
and later:
```
30s = Fitting data preprocessors runtime
13 features in original data, 15 features in preprocessed data
```

Reply:
Added 2nd half of comment, tried to add 'preprocess the data', but it looked messy.